cypress/support/constants.ts

```diff
@@ -22,7 +22,7 @@ const DEFAULT_GCP_SERVICEACCOUNT_PATH = Cypress.env('gcp_service_account_path');
 const DEFAULT_GCP_PROJECTID = Cypress.env('gcp_projectid');
 
 const DEFAULT_GCS_CONNECTION_NAME = `gcs_${getRandomArbitrary()}`;
-const DEFAULT_GCS_FOLDER = 'cdap-gcp-ui-test';
+const DEFAULT_GCS_FOLDER = '000cdap-gcp-ui-test';
```

Comment:
Why 000?

Reply:
Oh let me add a comment. The test bucket has been pushed down by other buckets so it doesn't appear in the listing.